docs/getting-started/index.md

```diff
@@ -28,34 +28,53 @@ under the License.
-  specific Flink concepts
-* explore on of the **Code Walkthroughs** if you want to get an end-to-end
-  introduction to using one of the Flink APIs
-* use **Project Setup** if you already know the basics of Flink but want to get a
-  project setup template for Java or Scala and need help setting up
-  dependencies
+* take a look at the **Docker Playgrounds** if you want to see what Flink can do, via a hands-on,
+  docker-based introduction to specific Flink concepts
+* explore one of the **Code Walkthroughs** if you want a quick, end-to-end
+  introduction to one of Flink's APIs
+* work your way through the **Hands-on Training** for a comprehensive,
+  step-by-step introduction to Flink
+* use **Project Setup** if you already know the basics of Flink and want a
+  project template for Java or Scala, or need help setting up the dependencies
 
 ### Taking a first look at Flink
 
 The **Docker Playgrounds** provide sandboxed Flink environments that are set up in just a few minutes and which allow you to explore and play with Flink.
 
-* The [**Operations Playground**](./docker-playgrounds/flink-operations-playground.html) shows you how to operate streaming applications with Flink. You can experience how Flink recovers application from failures, upgrade and scale streaming applications up and down, and query application metrics.
+* The [**Operations Playground**]({% link getting-started/docker-playgrounds/flink-operations-playground.md %}) shows you how to operate streaming applications with Flink. You can experience how Flink recovers application from failures, upgrade and scale streaming applications up and down, and query application metrics.
 
 <!-- 
 * The [**Streaming SQL Playground**]() provides a Flink cluster with a SQL CLI client, tables which are fed by streaming data sources, and instructions for how to run continuous streaming SQL queries on these tables. This is the perfect environment for your first steps with streaming SQL. 
 -->
 
 ### First steps with one of Flink's APIs
 
-The **Code Walkthroughs** are the best way to get started and introduce you step by step to an API.
-A walkthrough provides instructions to bootstrap a small Flink project with a code skeleton and shows how to extend it to a simple application.
+The **Code Walkthroughs** are a great way to get started quickly with a step-by-step introduction to
+one of Flink's APIs. Each walkthrough provides instructions for bootstrapping a small skeleton
+project, and then shows how to extend it to a simple application.
 
-* The [**DataStream API**](./walkthroughs/datastream_api.html) code walkthrough shows how to implement a simple DataStream application and how to extend it to be stateful and use timers. The DataStream API is Flink's main abstraction to implement stateful streaming applications with sophisticated time semantics in Java or Scala.
+* The [**DataStream API**]({% link getting-started/walkthroughs/datastream_api.md %}) code walkthrough shows how
```

Comment:
Should you actually also include "code walkthrough" in the link? (we're not linking to the DataStream API docs)
```suggestion
* The [**DataStream API**  code walkthrough]({% link getting-started/walkthroughs/datastream_api.md %}) shows how
```